js_modules/dagit/packages/core/src/app/Telemetry.tsx

```diff
@@ -1,18 +1,16 @@
-import {gql, useMutation, useQuery} from '@apollo/client';
-
-import {DagitTelemetryEnabledQuery} from './types/DagitTelemetryEnabledQuery';
+import {gql} from '@apollo/client';
+import {extractInitializationData} from '@dagit/app/src/extractInitializationData';
```

Comment:
`@dagit/app` may import from `@dagit/core`, but not vice versa. I should put a lint rule in place for that.